lib/box/tty.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright 2020 Gravitational, Inc.
```

Comment:
Is this file a complete copy-paste from runc source? If so, then we should probably use their license header.

Reply:
Yea, they don't have license headers in each file, so I've just removed it for now.